transpire/helpers/helm.py

```diff
@@ -117,7 +155,7 @@ def build_chart(
         f"ocf-{name}",
         "--api-versions",
         ", ".join(capabilities),
-        chart_name,
+        archive_name,
```

Comment:
I think you need to give it the full path to the archive, or it won't be able to find it.